src/utils/aepp-wallet-communication/rpc/aepp-rpc.js

```diff
@@ -151,6 +156,26 @@ export default Ae.compose({
       })
       return this.sendConnectRequest()
     },
+    /**
+     * Connect to node
+     * @function bridgeNode
+     * @instance
+     * @return {void}
+     */
+    async bridgeNode () {
+      if (!this.rpcClient || !this.rpcClient.isConnected()) throw new NoWalletConnectedError('You are not connected to Wallet')
```

Comment:
This line already repeated 7 times there, proposing to extract it.

Reply:
Sure